src/core/public/utils/index.ts

```diff
@@ -7,4 +7,16 @@
  */
 
 export { MountWrapper, mountReactNode } from './mount';
-export { KBN_LOAD_MARKS } from './consts';
+export {
+  KBN_LOAD_MARKS,
+  LOAD_START,
+  LOAD_CORE_CREATED,
+  KIBANA_LOADED_EVENT,
+  LOAD_SETUP_DONE,
+  LOAD_START_DONE,
+  LOAD_FIRST_NAV,
+  LOAD_BOOTSTRAP_START,
+} from './events';
```

Comment:
AFAIK, plugins can import these values as `src/core/public/utils` is an allowed import patch.

Probably, we should move this outside `utils`.